src/common/commandDescriptios.ts

```diff
@@ -0,0 +1,21 @@
+export const GENERAL_DESCRIPTION = "Greenhouse is a command-line tool that provides helpers to automate interactions with the Canonical Greenhouse website.";
+
+export const ADD_POSTS_USAGE = "([-i | --interactive] | <job-id> --regions=<region-name>[, <region-name-2>...] [--clone-from=<job-post-id>]) \n\n Examples: \n\t greenhouse add-posts --interactive \n \t greenhouse add-posts 1234 --regions=emea,americas \n \t greenhouse add-posts 1234 --regions=emea --clone-from=1123";
+
+export const ADD_POSTS_DESCRIPTION = "This command creates job posts for a specific job <job-id> in the specified regions <region-name> from all existing job posts in the Canonical Board. If the clone-from is specified, the command will create job posts in the specified regions from that job post. The command can be run interactively with the –interactive flag. This will prompt a set of questions for the user to answer.";
```

Comment:
This sentence sounds a bit repetitive to me:
"If the clone-from is specified, the command will create job posts in the specified regions from that job post"

Maybe something along the lines:
"If clone-from is specified, the new job post will be created based on the existing one"

(Can't come up with anything better right now)